tutorials/plot_sleep.py

```diff
@@ -0,0 +1,281 @@
+"""
+Example on sleep data
```

Comment:
You might want to be more specific on the title

```
Sleep prediction using XXX data
```

Reply:
I would go with 

Sleep stage classification from polysomnography (PSG) data